staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/errors/statuserror.go

```diff
@@ -32,6 +33,10 @@ func ToStatusErr(webhookName string, result *metav1.Status) *apierrors.StatusErr
 		result = &metav1.Status{Status: metav1.StatusFailure}
 	}
 
+	if result.Code < http.StatusBadRequest {
+		result.Code = http.StatusBadRequest
```

Comment:
what if we pass a status w/ "200" status.Code __and__ "success" status.Status? do we need to override the status to failure/unknown?

Reply:
I'll update these cases:
* if it's set to "Success", that's nonsensical, so we should override.
* if it's set to "", we can set to "Failure" (that happens later in the stack as well, but it doesn't hurt to set it here as well where we have more context)

if it's set to something other than "Success" or "Failure", I'm really hesitant to mess with it

Reply:
updated